src/main/scala/is/hail/annotations/Region.scala

```diff
@@ -20,7 +20,7 @@ object Region {
 final class Region(
   private var mem: Array[Byte],
   private var end: Long = 0
-) extends KryoSerializable with Serializable with AutoCloseable {
+) extends UnKryoSerializable with Serializable with AutoCloseable {
```

Comment:
will kryo serialization try to use normal java serialization if it can't do the kryo thing?